src/dex/solidly/forks-override/spiritSwapV2.ts

```diff
@@ -0,0 +1,60 @@
+import { Solidly } from '../solidly';
+import { SolidlyPair } from '../types';
+import { Network } from '../../../constants';
+import { IDexHelper } from '../../../dex-helper';
+import { Interface } from '@ethersproject/abi';
+import { SolidlyConfig } from '../config';
+import { getDexKeysWithNetwork } from '../../../utils';
+import _ from 'lodash';
+
+const SpiritSwapV2PairABI = [
+  {
+    inputs: [],
+    name: 'fee',
+    outputs: [{ internalType: 'uint256', name: '', type: 'uint256' }],
+    stateMutability: 'view',
+    type: 'function',
+  },
+];
+
```

Comment:
ABI may be passed on initialization part. And you can pass `variation: 'solidly' | 'spiritswapv2' | 'velodrome'` to make a difference in fee calculation. I think it may be more clean

Reply:
You can have that `variation` variable in config